### PR TITLE
JAMES-2302:SMTP configuration documentation refers to servername

### PR DESCRIPTION
**Pull request by Daniel-Dos**
PR with fix to item https://issues.apache.org/jira/browse/JAMES-2302

### CODE REVIEW

File: src/site/xdoc/server/config-smtp-lmtp.xml [hunk 1]
```diff
@@ -151,8 +151,8 @@
       
         <li>As mentioned above, SMTP AUTH requires that Apache James Server be able to distinguish between mail intended 
             for local delivery and mail intended for remote delivery.  Apache James Server makes this determination by matching the 
-            domain to which the mail was sent against the &lt;servernames&gt; element of the Apache James Server configuration block.  Any 
-            local domains should be explicitly listed as &lt;servername&gt; elements in this section.</li>
+            domain to which the mail was sent against the &lt;domainnames&gt; element of the Apache James Server configuration block.  Any
```
Comment (chibenwa) [inline src/site/xdoc/server/config-smtp-lmtp.xml]: You are right to show us this configuration is out-dated.

However, to be correct it should be:

```
to which the mail was sent against the &lt;DomainList&gt; component, configured in <a href="/server/config-domainlist.xml">domainlist.xml</a> configuration file.
```

Cheers

Comment (Daniel-Dos) [reply src/site/xdoc/server/config-smtp-lmtp.xml]: Great, I'll adjust it. thank you very much.

Comment (Daniel-Dos) [reply src/site/xdoc/server/config-smtp-lmtp.xml]: done. please review .

**Comment by Daniel-Dos** [comment]
your welcome  .

**Comment by Daniel-Dos** [comment]
@rouazana ,
could review this PR, it was just a change in the documentation.

File: src/site/xdoc/server/config-smtp-lmtp.xml [hunk 2]
```diff
@@ -152,8 +152,8 @@
       
         <li>As mentioned above, SMTP AUTH requires that Apache James Server be able to distinguish between mail intended 
             for local delivery and mail intended for remote delivery.  Apache James Server makes this determination by matching the 
-            domain to which the mail was sent against the &lt;servernames&gt; element of the Apache James Server configuration block.  Any 
-            local domains should be explicitly listed as &lt;servername&gt; elements in this section.</li>
+            domain to which the mail was sent against the &lt;DomainList&gt; component, configured in <a href="/server/config-domainlist.xml">domainlist.xml</a> configuration file. Any 
+            local domains should be explicitly listed as &lt;domainname&gt; elements in this section.</li>
```
Comment (mbaechler) [inline src/site/xdoc/server/config-smtp-lmtp.xml:156]: as far as I know, DomainList can be implemented with a database (like postgres with JPA or cassandra), so the wording may be misleading here.

Comment (rouazana) [reply src/site/xdoc/server/config-smtp-lmtp.xml:156]: yes, probably it should just point to the domain-list documentation page without stating about domainname elements which are implementation specific.

**Comment by rouazana** [comment]
and sorry to have forgotten this contribution!

**Comment by Daniel-Dos** [comment]
how can I improve this part of the documentation?

>

File: src/site/xdoc/server/config-smtp-lmtp.xml (hunk 2 above)
Comment (rouazana) [inline src/site/xdoc/server/config-smtp-lmtp.xml:156]:
```suggestion
            domain to which the mail was sent against the &lt;DomainList&gt; component, configured by <a href="/server/config-domainlist.html">Domain List Configuration</a>.
```

**Comment by rouazana** [comment]
> how can I improve this part of the documentation?
> […](#)

I suggested a simple fix

File: src/site/xdoc/server/config-smtp-lmtp.xml (hunk 2 above)
Comment (Daniel-Dos) [reply src/site/xdoc/server/config-smtp-lmtp.xml]: done .

**Comment by Daniel-Dos** [comment]
Hello @rouazana ,

Can review again ? 

thanks .

**Comment by Daniel-Dos** [comment]
thanks   :  )

**Comment by Daniel-Dos** [comment]
hello @chibenwa  

that ready for merge ?

**Comment by rouazana** [comment]
Hi,

This pull request has been merged.

One very last thing: I don't have write access to this repository, hence I can not close this issue. Wouldn't you mind doing it for me? It would avoid me annoying the Apache INFRA team with such simple concerns.

Thanks again,

Cheers,

Benoit

**Comment by Daniel-Dos** [comment]
thanks

**Pull request closed without merging**